aggregating-profiler/src/main/java/ru/fix/aggregating/profiler/engine/AggregatingReporter.java

```diff
@@ -78,8 +78,22 @@ private ProfilerReport buildReportAndReset(Optional<String> tagName,
         Map<String, Long> indicators = profiler.getIndicators()
                 .entrySet()
                 .stream()
-                .filter(entry -> ! tagName.isPresent() || entry.getValue().hasTag(tagName.get(), tagValue.orElse(null)))
-                .collect(Collectors.toMap(
+                .filter(entry -> ! tagName.isPresent()
+                        || entry.getValue().hasTag(tagName.get(), tagValue.orElse(null)))
+                .filter(entry -> {
+                        try {
+                            if(entry.getValue().getProvider().get() != null) {
+                                return true;
+                            }
+                            log.error("Inicator '{}' return null value", entry.getKey());
+                            return false;
```

Comment:
provider can return not null during first invocation and null value during second invocation. do not treat provider value as immutable - it can change during time.

Reply:
ok